### DISCUSSION TITLE
High CPU Usage

### DISCUSSION THREAD

**Discussion by mihnearad** [YimMenu/HorseMenu#207] [Q&A] [answered]
When injecting the .dll, CPU usage raises by about 20-25% compared to RDO alone on a 5900X. Is this expected behaviour? 

I'm using Xenos to inject.

**Comment by FIying-Scotsman** [comment, marked as answer]
I'd assume this is normal behaviour due to pointer scanning during initialisation and the pointer cache pull request [not being merged in](https://github.com/YimMenu/HorseMenu/pull/140) for whatever reason. Every time the menu loads it'll re-scan for the pointers instead of using cached values.

**Reply by Rxann** [reply]
Yes, I closed it because it was out of date with the codebase. There was faulty code pushed before and somehow it got into that code. I didn't feel like dealing with it and I would have had to basically rewrite it to implement the part of storing sig lengths.

**Reply by mihnearad** [reply]
Thanks, it's clear. However, it kept being high even 10 minutes into the game. Perhaps need to give it more time. Thank you for the clarity.

**Reply by Rxann** [reply]
Your welcome, let us know if it persists. RDR2 is a resource intensive game keep in mind.
